<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodePluginSelectionPage</class>
 <widget class="QWidget" name="NodePluginSelectionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_title">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Create a Node Plugin</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="gbx_NodeOptions">
       <property name="title">
        <string>Node Plugin Options</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="0" colspan="2">
         <widget class="QCheckBox" name="cbx_addObjs">
          <property name="text">
           <string>Add objects</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QComboBox" name="cmbo_nodeType"/>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="lbl_className">
          <property name="text">
           <string>Class Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLineEdit" name="line_className">
          <property name="placeholderText">
           <string>Enter name for Node class</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="lbl_nodeType">
          <property name="text">
           <string>Select Node type to make</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
